app/talk/comment-box.cjsx

```diff
@@ -44,68 +43,31 @@ module?.exports = React.createClass
 
   onSubmitComment: (e) ->
     e.preventDefault()
-    textareaValue = @refs.textarea.getDOMNode().value
+    textareaValue = @state.content
     return if @props.validationCheck?(textareaValue)
     @setState loading: true
     fullComment = @state.reply.concat(textareaValue)
 
     @props.onSubmitComment?(e, fullComment, @state.subject)
       .then =>
-        @refs.textarea.getDOMNode().value = ''
         @hideChildren()
-        @setState content: '', error: '', loading: false
+        @setState subject: null, content: '', error: '', loading: false
         @setFeedback @props.submitFeedback
       .catch (e) =>
         @setState(error: e.message, loading: false)
 
-  onPreviewClick: (e) ->
-    @toggleComponent('preview')
-
-  onHelpClick: (e) ->
-    @toggleComponent('help')
+  onInputChange: (e) ->
```

Comment:
It seems like some button is acting as `type=submit` since that shouldn't be necessary.